content/BCPatterns/event-bridge-pattern/index.md

```diff
@@ -0,0 +1,115 @@
++++
+title = "Event Bridge Pattern"
+weight = 1180
++++
+
+<_Created by waldo & Arend-Jan Kauffmann, Described by waldo_\>
+
+## Abstract
+
+In the world of interfaces, it is important to preserve (certain) events over multiple implementation of the interface.
+
+## Context
+
+An app can have interfaces.
+It makes it possible for other apps to extend/change the implementations of a certain part of the business logic.
+
+Like in this example, we have an interface, to implement different ways for getting weights from scales:
+
+```AL
+interface "IScale"
+{
+    procedure GetWeight(): Decimal;
+    procedure Tare();
```

Comment:
Q: Is a second procedure neccessary for describing this pattern or wouldn't it be simpler to just focus on one?
In any case, it shows that the pattern is applied consistenlty, which is good.

Reply:
It isn't - but it made it easy for me to copy/paste ;-).  Let's keep it?

Reply:
Sure let's keep them 👍